elvis.config

```diff
@@ -38,6 +38,7 @@
                                                                    , {right, "=/="}
                                                                    , {left , "=/="}
                                                                    ]}}
+                      , {elvis_style, function_naming_convention, #{ignore => [els_methods]}}
```

Comment:
@jfacorro Is there a way to allow a different regex only for a given module? In this case I would like to allow the `$` for the `els_methods` module...

Reply:
Not Juan, but I can answer this one. It is currently not possible. You can see the implementation here: https://github.com/inaka/elvis_core/blob/master/src/elvis_style.erl#L158

It's something that I've had on my personal projects list to fix 👍 I'll see if I can spend some time on the weekend to allow for a per module regex 👍

Reply:
As @onno-vos-dev said, sadly there's not. 😞

Reply:
The `els_method` module doesn't have any functions whose name starts iwth `$_` so I think this ignore could be removed, right?

Reply:
Yes, this line is a leftover from a previous commit where I had individual functions for the supported `$/` notifications.